setup.cfg

```
@@ -77,7 +77,7 @@ install_requires =
     colorama == 0.3.9
     cookiecutter == 1.6.0
     python-gilt >= 1.2.1, < 2
-    Jinja2 == 2.10
+    Jinja2 == 2.10.1
```

Can we please have an open upper boundary?
```suggestion
    Jinja2 >= 2.10.1
```